addons/website_slides/models/slide_channel_tag.py

```diff
@@ -12,6 +12,7 @@ class SlideChannelTagGroup(models.Model):
     name = fields.Char('Name', required=True, translate=True)
     sequence = fields.Integer('Sequence', default=10, index=True, required=True)
     tag_ids = fields.One2many('slide.channel.tag', 'group_id', string='Tags')
+    # TODO: use mixin? -> Then change 6 occurence in <record> and some in form/tree view
     website_published = fields.Boolean(
```

Comment:
We should use mixin here, the copy=True can now be overrided of the mixin field (see previous commit).
And the help is not that important TBH

Reply:
I think it would be best to rename the field to `menu_entry` and keep it like this (it is used only once in a search), to keep the `Menu entry` string and `Makes a menu entry in main navigation of Slides, allowing to filter on its tags directly from main navigation.'` help.

Reply:
(I didn't really check the help/string but it is really different from the ones of the mixin)

Reply:
using the mixin allow to add the publish button stat and co. Why not use it ? ,  ,

Reply:
Done, used `string` and `help` to keep what was done before.
Functionnaly, only change is the button stat replace the boolean.
Technically, we now use the mixin.